astropy/io/fits/util.py

```diff
@@ -292,7 +292,7 @@ def decode_ascii(s):
         if s.size == 0:
             # Numpy apparently also has a bug that if a string array is
             # empty calling np.char.decode on it returns an empty float64
-            # array wth
+            # array with
```

Comment:
This comment seems incomplete, but it's likely that "with" is the word that was intended to be here.

Reply:
You sure it isn't "w.t.h."? 😆

Reply:
I guess those random letters can be removed :D 
I checked and the numpy issue is still valid: https://github.com/numpy/numpy/issues/13156, we could mention the number in the comment.